### PR TITLE
Creating print leader board

### PR DESCRIPTION
**Pull request by paulaOliverosHollis**

### CODE REVIEW

File: DeveloperGame/DeveloperGame/ComputerGeekGame.cs [hunk 1]
```diff
@@ -80,8 +80,7 @@ private void ReadScoresFromFile()
 
         private void PrintIntro()
         {
-            Console.WriteLine("\nWelcome To Computer Geek Game!");
-            Console.WriteLine("\nIn this game, your knowledge about Computer Science history is going to be tested. The test cosists of multiple choice and true or false questions. " +
+            Console.WriteLine("\nWelcome To Computer Geek Game!\nIn this game, your knowledge about Computer Science history is going to be tested. The test cosists of multiple choice and true or false questions. " +
```
Comment (zhollis21) [inline DeveloperGame/DeveloperGame/ComputerGeekGame.cs]: Before you had 2 new lines, now you have 1

File: DeveloperGame/DeveloperGame/ComputerGeekGame.cs [hunk 2]
```diff
@@ -142,10 +141,12 @@ private void Play()
 
         private void PrintLeaderBoard()
         {
-           foreach(var x in _leaderBoard)
-           {
-                Console.WriteLine($"{x.PlayerName} {x.Points} {x.Time}");
-           }
+            Console.WriteLine("\n\t|| Leader Board ||\n");
+
+            foreach (var x in _leaderBoard)
```
Comment (zhollis21) [inline DeveloperGame/DeveloperGame/ComputerGeekGame.cs]: x is no a good variable name

**Review by zhollis21 — CHANGES_REQUESTED** [review]

File: DeveloperGame/DeveloperGame/ComputerGeekGame.cs (hunk 1 above)
Comment (paulaOliverosHollis) [reply DeveloperGame/DeveloperGame/ComputerGeekGame.cs]: Good, you know how to count.

File: DeveloperGame/DeveloperGame/ComputerGeekGame.cs (hunk 2 above)
Comment (paulaOliverosHollis) [reply DeveloperGame/DeveloperGame/ComputerGeekGame.cs]: Zach is no good name.

File: DeveloperGame/DeveloperGame/Score.cs [hunk 3]
```diff
@@ -4,53 +4,23 @@
...
-            set
-            {
-                _time = value;
-            }
-        }
+        public int Points { get; set; }
+
+        public TimeSpan Time { get; set; }
 
         public Score(string playerName, int points, TimeSpan time)
```
Comment (zhollis21) [inline DeveloperGame/DeveloperGame/Score.cs]: You forgot to change this

**Review by zhollis21 — APPROVED** [review]
:fu:

**Pull request merged**